buildres/upload-release.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# This script is used to upload signed and notarized releases to S3 and update the Electron auto-update release feeds.
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+BUILDS_DIR=$SCRIPT_DIR/builds
+TEMP2_DIR=$SCRIPT_DIR/temp2
+
+# Copy the builds to the temp2 directory
+echo "Copying builds to temp2"
+rm -rf $TEMP2_DIR
+mkdir -p $TEMP2_DIR
+cp -r $BUILDS_DIR/* $TEMP2_DIR
+
+UVERSION=$(cat $TEMP2_DIR/version.txt)
+
+if [ -z "$UVERSION" ]; then
+    echo "version.txt is empty"
+    exit 1
+fi
+
+# Find the DMG file
+echo "Finding DMG"
+DMG=$(find $TEMP2_DIR -type f -iname "*.dmg")
+# Ensure there is only one
+NUM_DMGS=$(echo $DMG | wc -l)
+if [ "0" -eq "$NUM_DMGS" ]; then
+    echo "no DMG found in $TEMP2_DIR"
+    exit 1
+elif [ "1" -lt "$NUM_DMGS" ]; then
+    echo "multiple DMGs found in $TEMP2_DIR"
+    exit 1
+fi
+
+# Find the Mac zip
+echo "Finding Mac zip"
+MAC_ZIP=$(find $TEMP2_DIR -type f -iname "*mac*.zip")
+# Ensure there is only one
+NUM_MAC_ZIPS=$(echo $MAC_ZIP | wc -l)
+if [ "0" -eq "$NUM_MAC_ZIPS" ]; then
+    echo "no Mac zip found in $TEMP2_DIR"
+    exit 1
+elif [ "1" -lt "$NUM_MAC_ZIPS" ]; then
+    echo "multiple Mac zips found in $TEMP2_DIR"
+    exit 1
+fi
+
+# Find the Linux zips
+echo "Finding Linux zips"
+LINUX_ZIPS=$(find $TEMP2_DIR -type f -iname "*linux*.zip")
+# Ensure there is at least one
+NUM_LINUX_ZIPS=$(echo $LINUX_ZIPS | wc -l)
+if [ "0" -eq "$NUM_LINUX_ZIPS" ]; then
+    echo "no Linux zips found in $TEMP2_DIR"
+    exit 1
+fi
+
+# Upload the DMG
+echo "Uploading DMG"
+DMG_NAME=$(basename $DMG)
+aws s3 cp $DMG s3://dl.waveterm.dev/$DMG_NAME
```

Comment:
Right now the (non-autoupdate) builds are in s3://dl.waveterm.dev/builds/* , so I think these s3 paths should add /build/.  (same for the linux zips below).

Reply:
This is now fixed